src/utils/data-fetcher-airtable.js

```diff
@@ -38,5 +39,46 @@ module.exports = {
 				}
 			});
 		});
+	},
+	populateResources: async () => {
+		return new Promise((resolve, reject) => {
+			const tagLabelToValue = {};
+			learnTags.resourceTags.forEach(tag => {
+				tagLabelToValue[tag.label] = tag.value;
+			});
+			const resources = [];
+			// Need Airtable table id for resources
+			base(env.airtableTableId).select().eachPage(function page(records, fetchNextPage) {
```

Comment:
Is there a reason to define the airtable base name as a confidential environment variable for this one time migration? It should be OK to use the name directly here.

Defining as an environment variable requires a change to the deploy settings. Otherwise, this error is thrown: `Table name or table ID is required`.